source/features/index.tsx

```diff
@@ -148,23 +148,48 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 	resolve(options);
 });
 
+function setupDeinit(deinit: Deinit): void {
```

Comment:
Can we change this function to this since `addEventListener` is repeated 4 times here?

```suggestion
function setupDeinit(deinit: Deinit): void {
	document.addEventListener('pjax:start', getDeinitHandler(deinit), {once: true});
}
```